ui/src/App.scss

```diff
@@ -5,3 +5,11 @@
   overflow: hidden;
   position: fixed;
 }
+.footer {
```

Comment:
Why do we declare two classes of .footer. We declare one here and the other in footer.scss

Reply:
Fixed and removed